.gitignore

```diff
@@ -1,10 +1,74 @@
-*.iml
-.gradle
-/local.properties
-/.idea
 .DS_Store
+
+.gradle
+**/build/
+!src/**/build/
 /build
+.gradle/
+build/
+
+# Ignore Gradle GUI config
+gradle-app.setting
+
+# Cache of project
+.gradletasknamecache
+
+# Eclipse Gradle plugin generated files
+# Eclipse Core
+.project
+# JDT-specific (Eclipse Java Development Tools)
+.classpath
+
+# Compiled class file
+*.class
+
+# BlueJ files
+*.ctxt
+
+# Mobile Tools for Java (J2ME)
+.mtj.tmp/
+
+# Package Files #
+*.jar
+*.war
+*.nar
+*.ear
+*.zip
+*.tar.gz
+*.rar
+
+# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
+hs_err_pid*
+replay_pid*
+
+# Local configuration file (sdk path, etc)
+local.properties
+/local.properties
+
+# Log/OS Files
+*.log
+
+# Android Studio generated files and folders
+captures/
 /captures
+.externalNativeBuild/
 .externalNativeBuild
+.cxx/
 .cxx
-local.properties
+*.apk
+output.json
+
+# IntelliJ
+*.iml
+.idea/
+/.idea
+misc.xml
+deploymentTargetDropDown.xml
+render.experimental.xml
+
+# Keystore files
+*.jks
+*.keystore
+
+# Android Profiling
+*.hprof
```

Comment:
Could you please add a new line at the end of the file?

**Reason:** https://unix.stackexchange.com/a/18746

Reply:
sure, done ✅